lib/request.d.ts

```diff
@@ -54,23 +54,23 @@ export class Request<D, E> {
      * @param {function} listener - Callback to run when the request is being validated.
      * @param {boolean} prepend - If set, prepends listener instead of appending.
      */
-    on(event: "validate", listener: (request: Request<D, E>) => void, prepend?: boolean): Request<D, E>;
+    on(event: "validate", listener: (request: Request<D, E>, doneCallback?: () => void) => void, prepend?: boolean): Request<D, E>;
```

Comment:
I added `doneCallback?` as an optional second param, trying to be sensitive to not breaking anybody's current code that may not use the param. Although I'm divided on whether or not it should probably actually be a required argument.